src/components/form/range/range.tsx

```diff
@@ -116,6 +116,26 @@ export class EuiRangeClass extends Component<
     });
   };
 
+  handleAriaValueText = (
+    ticks: EuiRangeTick[],
+    currentVal: string | number
+  ): string | undefined => {
+    let ariaValueText;
+    const target = ticks.find(
+      (tick) => tick.value.toString() === currentVal.toString()
+    );
+
+    if (target) {
+      ariaValueText = target?.accessibleLabel
+        ? `${target.value.toString()}, (${target.accessibleLabel})`
+        : target.value.toString();
+
+      return ariaValueText;
+    }
+
+    return ariaValueText; // undefined
```

Comment:
I don't think we need the `let` or all these returns. (there's no need to return at the end for an undefined, that's already implied). Let's simplify:

```suggestion
    const target = ticks.find(
      (tick) => tick.value.toString() === currentVal.toString()
    );

    if (target) {
      return target.accessibleLabel
        ? `${target.value.toString()}, (${target.accessibleLabel})`
        : target.value.toString();
    }
```

And if we don't need the ternary fallback (see above comments), then we simplify even further:

```suggestion
    const target = ticks.find(
      (tick) => tick.value.toString() === currentVal.toString()
    );

    if (target && target.accessibleLabel) {
      return `${target.value.toString()}, (${target.accessibleLabel})`
    }
```